pkg/admin/server/istio.go

```diff
@@ -13,15 +13,16 @@ import (
 )
 
 const (
-	CDS_UPDATE_SUCCESS = "cluster_manager.cds.update_success"
-	CDS_UPDATE_REJECT  = "cluster_manager.cds.update_rejected"
-	LDS_UPDATE_SUCCESS = "listener_manager.lds.update_success"
-	LDS_UPDATE_REJECT  = "listener_manager.lds.update_rejected"
-	SERVER_STATE       = "server.state"
+	CDS_UPDATE_SUCCESS   = "cluster_manager.cds.update_success"
+	CDS_UPDATE_REJECT    = "cluster_manager.cds.update_rejected"
+	LDS_UPDATE_SUCCESS   = "listener_manager.lds.update_success"
+	LDS_UPDATE_REJECT    = "listener_manager.lds.update_rejected"
+	SERVER_STATE         = "server.state"
+	STAT_WORKERS_STARTED = "listener_manager.workers_started"
 )
 
 func statsForIstio(w http.ResponseWriter, r *http.Request) {
-	state, err := getIstioState()
+	state, workersStarted, err := getIstioState()
```

Comment:
set the value of `workersStarted` could  via state.

Reply:
already reset getIstioState method